jest.config.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
```

Comment:
This file shoule be placed in `tests/` directory (or `test/js/`). You should be able to do this by using the `--config` option in jest command call.